.eslintrc.json

```diff
@@ -234,7 +264,13 @@
                     "off"
                 ],
                 "no-await-in-loop": [
-                    "off"
+                    "error"
+                ],
+                "no-return-await": [
+                    "error"
+                ],
+                "no-promise-executor-return": [
+                    "error"
                 ],
```

Comment:
For the [promise](https://cnpmjs.org/package/eslint-plugin-promise) plugin, I'd set following:

```
{
  "rules": {
    "promise/always-return": "error",
    "promise/no-return-wrap": "off",
    "promise/param-names": "error",
    "promise/catch-or-return": "error",
    "promise/no-native": "off",
    "promise/no-nesting": "off",
    "promise/no-promise-in-callback": "warn" / "error",
    "promise/no-callback-in-promise": "off",
    "promise/avoid-new": "off",
    "promise/no-new-statics": "error",
    "promise/no-return-in-finally": "warn",
    "promise/valid-params": "warn"
  }
}
```

Reply:
I am not sure if warn is useful, as most of checks are run on CI pipelines they won't be seen, and only errors are actionable.
So I would vote to put both last 2 options as error